GitUI/UserControls/FileStatusList.cs

```diff
@@ -998,69 +1014,86 @@ void EnsureSelectedIndexChangeSubscription()
 
             int GetItemImageIndex(GitItemStatus gitItemStatus)
             {
+                // If imageKey does not exist, trow an exception, implementation error
```

Comment:
Using a mobile app, and couldn't see if we throw the exception. 
Btw,  typo "throw".

Instead throwing an exception it is better to enforce via a test.

Reply:
Check with ContainsKey